src/App.js

```diff
@@ -1,23 +1,19 @@
-import logo from './logo.svg';
 import './App.css';
 
+// import background from './background.jpg';
+
+import Header from './components/Header/Header.js';
+import Footer from './components/Footer/Footer.js';
+import Main from './components/Main/Main';
+
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      {/* style={{ backgroundImage: `url(${background})` }}> */}
+
+      <Header />
+      <Main />
+      <Footer />
     </div>
   );
```

Comment:
I believe the zodiac signs overfill into the header because they are all in the same div within App.js. If you create three different div's wrapped in <> </>, I think this will fix the issue :)